### DISCUSSION TITLE
Cant get converter to work

### DISCUSSION THREAD

**Discussion by BackHive** [thewsoftware/sdrfc#8] [Q&A] [answered]
Hi there,
I really like the idea of this tool, so many thanks for your work. It would be of great use to me if I could get it to work.
When I create the file for SDR++ from my rather large SDR# file, rename it to 'frequency_manager_config.json' and place it in the SDR++ folder, SDR++ simply replaces it with it's startup default 1kb 'frequency_manager_config.json' file (which of course contains no frequency entries). I removed any special characters from my SDR# file, tried it again and it does the same. Any advice appreciated. Many thanks.
Terry

**Comment by HuubH** [comment]
Hi,

For what it's worth. I had the same problem. It turned out I had a lot of double entries in de naming of frequency locations in SDR#. I used Visual Studio Code to analyse the problem. Edited the converted frequency list to see what would solve the problem. In the end (with no warnings in VSC anymore) the list loaded perfectly.

Huub

**Reply by thewsoftware** [reply, marked as answer]
Thank you very much for the feedbacks! 
To @BackHive, was SDR++ running when you placed the new frequency_manager_config.json into SDR++ folder?
Both SDR++ and SDR# load frequency files when started, manage it in the memory and overwrite them on closing the app. It appears to be the case. I would recommend to make sure the app is not running when replacing the frequency file. Hope it will help!

**Reply by thewsoftware** [reply]
@HuubH, double entries in frequency files should be converted as is when "-c" option is used. This mode is used by default. This option is helpful when, for example channels are managed in SDR++ and periodically getting converted to SDR#.
 
When "-m" option is chosen, duplicate frequency channels are combined into one, so there is always a unique entry for each combination of Group Name and Frequency. This mode is helpful when frequency channels are actively managed in both SDR++ and SDR# and converted back end forth. In this case the trade off is loosing the ability to have duplicate entries.
